src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1341,6 +1341,64 @@
         az acr token update -n MyToken -r MyRegistry --scope-map MyNewScopeMap
 """
 
+helps['acr artifact-streaming'] = """
+type: group
+short-summary: Manage artifact streaming in any repositories or images.
+"""
+
+helps['acr artifact-streaming operation'] = """
+type: group
+short-summary: Allows you to manage the streaming artifact creation.
+"""
+
+helps['acr artifact-streaming operation cancel'] = """
+type: command
+short-summary: Cancels the given operation.
```

Comment:
NIT: Cancels the given streaming artifact operation for Azure Container Registry.